unittests/FEXLinuxTests/tests/fd/test_close_range.cpp

```diff
@@ -0,0 +1,14 @@
+#include <cstdint>
+#include <unistd.h>
+
+int main() {
+  int fd_base = dup(STDOUT_FILENO);
+  for (size_t i = 0; i < 15; ++i) {
+    dup(fd_base);
+  }
+
+  // Specifically testing last as ~0U to ensure FEX doesn't hang
+  constexpr uint32_t SYS_close_range = 436;
+  ::syscall(SYS_close_range, fd_base, ~0U, 0);
```

Comment:
How about closing `fb_base+1` and then assert that 1) close(fd_base) succeeds since it wasn't closed by close_range 2) close (fd_base+1) fails since it's already closed?

Reply:
Sure thing. Switched over to that.